CHANGELOG.md

```diff
@@ -23,39 +24,53 @@
 
 ### New
 
-- Add New Relic Scaler ([#2387](https://github.com/kedacore/keda/pull/2387))
+- TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
+
+### Improvements
+
+- TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
+
+### Breaking Changes
+
+- TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
+
+### Other
+
+- TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
+
+## v2.6.0
+
+### New
+
 - Add ActiveMQ Scaler ([#2305](https://github.com/kedacore/keda/pull/2305))
+- Add Azure Application Insights Scaler ([2506](https://github.com/kedacore/keda/pull/2506))
 - Add New Datadog Scaler ([#2354](https://github.com/kedacore/keda/pull/2354))
+- Add New Relic Scaler ([#2387](https://github.com/kedacore/keda/pull/2387))
 - Add PredictKube Scaler ([#2418](https://github.com/kedacore/keda/pull/2418))
-- Add Azure Application Insights Scaler ([2506](https://github.com/kedacore/keda/pull/2506))
 
 ### Improvements
 
-- **General:** `keda-operator` Cluster Role: add `list` and `watch` access to service accounts ([#2406](https://github.com/kedacore/keda/pull/2406))|([#2410](https://github.com/kedacore/keda/pull/2410))
+- **AWS Scalers:** Support temporary AWS credentials using session tokens ([#2573](https://github.com/kedacore/keda/pull/2573))
+- **AWS SQS Scaler:** Allow using simple queue name instead of URL ([#2457](https://github.com/kedacore/keda/pull/2457))
+- **Azure EventHub Scaler:** Don't expose connection string in metricName ([#2404](https://github.com/kedacore/keda/pull/2404))
+- **Azure Pipelines Scaler:** Support `poolName` or `poolID` validation ([#2370](https://github.com/kedacore/keda/pull/2370))
+- **CPU Scaler:** AAding e2e test for the cpu scaler ([#2441](https://github.com/kedacore/keda/pull/2441))
+- **External Scaler:** Fix wrong calculation of retry backoff duration ([#2416](https://github.com/kedacore/keda/pull/2416))
 - **General:** Delete the cache entry when a ScaledObject is deleted ([#2564](https://github.com/kedacore/keda/pull/2564))
```

Comment:
I think that it would be better if we put `General` to the beginning or end of the list.

Reply:
Agreed, that was my intention with the PR indeed

Reply:
So general at the top, and behind it the other alphabetically sorted, do you agree?